### DISCUSSION TITLE
Diarrhea husks

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14625] [Unstable feedback]
Husks are now sporting the iconic brown ~~tentacles~~ crab legs

It's weird that husks look so brown and desaturated now considering every piece of official art and every fanart has bright red tentacles that contrast nicely with the rusted suit and cadaverous disposition of the host

Poor husk container looks like it just shat itself 

![image](https://github.com/user-attachments/assets/2520aa6f-cb24-4c8e-948d-18aa5f4b3fad)
![image](https://github.com/user-attachments/assets/61ac9991-5d3d-4606-ae09-038417767468)

![image](https://github.com/user-attachments/assets/6ffb1d8e-05bb-447e-b6a7-2a7f4d62330e)
![image](https://github.com/user-attachments/assets/0a5f557e-6094-4c18-a705-6b1ab9e4c06a)
![image](https://github.com/user-attachments/assets/73a814f6-bb9f-4a84-935d-f4387ba43e87)

**Comment by EebleSheembl** [comment]
While I don't really mind the "crabbiness" I do agree that it should stay consistent with the already made art, and I did some changes to the sprites myself to make them more like it and it does look better, to me at least

![image](https://github.com/user-attachments/assets/cf92554d-bc91-4b05-abb7-0d9bd261b681)

![image](https://github.com/user-attachments/assets/c98661ef-f3b8-4e98-8749-3618d4c5bef3)

![image](https://github.com/user-attachments/assets/c183cdec-6a48-4848-b4d0-ea2bcc10ce44)

![image](https://github.com/user-attachments/assets/ffefb5ae-ded1-4db1-adb0-fd402b753603)

![image](https://github.com/user-attachments/assets/4610acc1-ad2c-448e-b781-3f5488a5b1d4)

**Comment by Regalis11** [comment]
This was actually something I've wanted to change for a long time: according to all of the descriptions in-game, they're not tentacles: they're hard, chitinous ovipositors that they use to inject eggs into hosts. I've just always felt that the idea of what look like huge insect legs coming out of someone's mouth is a lot more unpleasant and original than slimy tentacles, and been bothered about it being retconned into tentacles in the art (despite all the texts in the game saying otherwise). At least personally I feel the current sprites are better even though they don't match what's been depicted in some of the existing art.

**Reply by ubertpendragon** [reply]
Suspected the "chitinous" bit of text being the reason and I think the game has been confused about that part 
![image](https://github.com/user-attachments/assets/b8dfeff2-7103-40e5-a548-1ae54d84d123)

There's a real problem with how it looks on unstable though. It's brown on brown

**Reply by EebleSheembl** [reply]
Well chitin can be multiple different colors not just... that color, like red

**Reply by Regalis11** [reply]
Yeah, I don't really have any strong opinions on the color, I just think they should look like insect legs, not tentacles. I suppose I could ask the art team how they feel about readjusting the color.
